### PR TITLE
Components to pinia

### PR DESCRIPTION
**Pull request by ssali-on-sparkz-systems**
upgrading converting components to vue 3 and pinia

### CODE REVIEW

File: src/components/banks/MyWallets.vue [hunk 1]
```diff
@@ -39,55 +39,53 @@
...
-      }),
-    ],
-    mounted() {
-      console.log('this is accountData in mounted: ', this.accountData);
-    },
-    data() {
+        params: {
+          debounce: 500,
+        }
+      });
```
Comment (dallinbjohnson) [inline src/components/banks/MyWallets.vue:74]: need to use computed and ref

**Review by dallinbjohnson — CHANGES_REQUESTED** [review]

File: src/components/banks/MyWallets.vue [hunk 2]
```diff
@@ -39,55 +39,53 @@
...
       return {
-        isLoading: false,
-        formData: undefined,
-        newWalletDio: false,
-        accountData: {
-          account: new models.api.Accounts().clone(),
-        },
+        wallets,
+        isPending,
+        isLoading: ref(false),
```
Comment (dallinbjohnson) [inline src/components/banks/MyWallets.vue]: what is `isLoading` being used for?

Comment (ssali-on-sparkz-systems) [reply src/components/banks/MyWallets.vue]: removed ... was not doing anything

File: src/components/banks/MyWallets.vue (hunk 1 above)
Comment (ssali-on-sparkz-systems) [reply src/components/banks/MyWallets.vue:74]: resolved

File: src/components/formGeneratorCustom/PlacesAutoCompleteBox/PlacesAutoComplete.vue [hunk 3]
```diff
@@ -110,20 +96,41 @@
       },
     },
     computed: {
-      ...mapState('places-auto-complete', {placeloading: 'isFindPending'}),
-      ...mapState('geocode', {geoloading: 'isFindPending'}),
-      ...mapGetters('places-auto-complete', {addresses: 'list'}),
+      ...mapState(usePlacesAutoComplete, {placeloading: 'isFindPending'}),
+      ...mapState(useGeocode, {geoloading: 'isFindPending'}),
+      ...mapState(usePlacesAutoComplete, {addresses: 'items'}),
```
Comment (dallinbjohnson) [inline src/components/formGeneratorCustom/PlacesAutoCompleteBox/PlacesAutoComplete.vue]: you did `mapState(usePlacesAutoComplete` twice combine them.

File: src/components/formGeneratorCustom/PlacesAutoCompleteBox/PlacesAutoComplete.vue [hunk 4]
```diff
@@ -110,20 +96,41 @@
...
+        this.$lset(newVal, 'div-attrs.class', this.$lget(newVal, 'div-attrs.class', 'col-12 col-sm-6'));
+        return newVal;
+      },
     },
     methods: {
-      ...mapMutations('places-auto-complete', {clearAddresses: 'clearAll'}),
-      ...mapActions('places-auto-complete', {findAddresses: 'find'}),
-      ...mapActions('geocode', {findgeocoded: 'find'}),
-      async clearInput() {
+      // ...mapMutations('places-auto-complete', {clearAddresses: 'clearAll'}),
```
Comment (dallinbjohnson) [inline src/components/formGeneratorCustom/PlacesAutoCompleteBox/PlacesAutoComplete.vue:127]: There is a `clearAll` action in the state!

File: src/components/formGeneratorCustom/PlacesAutoCompleteBox/PlacesAutoComplete.vue (hunk 3 above)
Comment (ssali-on-sparkz-systems) [reply src/components/formGeneratorCustom/PlacesAutoCompleteBox/PlacesAutoComplete.vue]: oh... resolved

File: src/components/formGeneratorCustom/PlacesAutoCompleteBox/PlacesAutoComplete.vue (hunk 4 above)
Comment (ssali-on-sparkz-systems) [reply src/components/formGeneratorCustom/PlacesAutoCompleteBox/PlacesAutoComplete.vue:127]: resolve

File: src/pages/HostsPage.vue [hunk 5]
```diff
@@ -15,67 +15,59 @@
...
 </template>
 
 <script>
   // import DataTableTemplate from '../components/common/substance/tables/DataTableTemplate';
   import {date} from 'quasar';
   import FDataTable from '../components/common/molecules/feathers/FDataTable/FDataTable';
   import {capitalize, kebabize} from '../utils';
-  import {makeFindPaginateMixin} from '../';
+  import {useFindPaginate} from '../';
+  import {computed, ref} from 'vue/dist/vue';
```
Comment (dallinbjohnson) [inline src/pages/HostsPage.vue]: `import {computed, ref} from 'vue/dist/vue';` to `import {computed, ref} from 'vue';`

Comment (ssali-on-sparkz-systems) [reply src/pages/HostsPage.vue]: resolved

File: src/store/services/places-auto-complete.js [hunk 6]
```diff
@@ -0,0 +1,106 @@
+import { defineStore, BaseModel } from 'feathers-pinia';
+import {hookCustomizer,lodash} from '../../';
+
+const {$lisNil, $lmergeWith} = lodash;
+
+
+Array.prototype.insert = function (index, ...value) {
+  this.splice(index, 0, ...value);
+  return this;
+};
```
Comment (dallinbjohnson) [inline src/store/services/places-auto-complete.js]: what is this being used for?

Comment (ssali-on-sparkz-systems) [reply src/store/services/places-auto-complete.js]: removed

**Review by dallinbjohnson — APPROVED** [review]

**Pull request merged**